src/components/cc-env-var-form/cc-env-var-form.stories.js

```diff
@@ -153,9 +153,12 @@ export const saving = makeStory(conf, {
     {
```

Comment:
**fix:** general feedback about the `no-context` story, I'm not sure it's something you wanted but the `cc-toggle` is no longer centered within this story:
Prod:
![image](https://github.com/user-attachments/assets/c336ffdc-79b6-425a-b55a-7679b216a518)


Preview:
![image](https://github.com/user-attachments/assets/4a2d87dc-72a1-4f01-b516-2dd4aca215ae)